telnet.c

```diff
@@ -95,6 +95,7 @@ struct ios_ops *telnet_init(char *hostport)
 	ios->set_speed = telnet_set_speed;
 	ios->set_flow = telnet_set_flow;
 	ios->send_break = telnet_send_break;
+	ios->istelnet = true;
```

Comment:
`serial.c` and `can.c` allocate `ios` with `malloc` and thus leave this member uninitialized.

Reply:
good catch. I fixed that in 36e939f.